service/src/test/java/org/cbioportal/service/impl/StructuralVariantEnrichmentServiceImplTest.java

```diff
@@ -0,0 +1,80 @@
+package org.cbioportal.service.impl;
+
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+import org.cbioportal.model.AlterationEnrichment;
+import org.cbioportal.model.MolecularProfileCaseIdentifier;
+import org.cbioportal.service.StructuralVariantService;
+import org.cbioportal.service.util.AlterationEnrichmentUtil;
+import org.junit.Assert;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.mockito.InjectMocks;
+import org.mockito.Mock;
+import org.mockito.Mockito;
+import org.mockito.junit.MockitoJUnitRunner;
+
+@RunWith(MockitoJUnitRunner.Silent.class)
+public class StructuralVariantEnrichmentServiceImplTest extends BaseServiceImplTest {
+
+    @InjectMocks
+    private StructuralVariantEnrichmentServiceImpl structuralVariantEnrichmentService;
+
+    @Mock
+    private StructuralVariantService structuralVariantService;
+    @Mock
+    private AlterationEnrichmentUtil alterationEnrichmentUtil;
```

Comment:
Any reason you're not diamond typing this? Makes code completion a little easier in some IDEs

```suggestion
    private AlterationEnrichmentUtil<StructuralVariantCountByGene> alterationEnrichmentUtil;
```